array.c

```diff
@@ -7764,7 +7764,7 @@ rb_ary_drop_while(VALUE ary)
  *  <b>Special case</b>: returns +false+ if +self+ is empty
  *  (regardless of any given argument or block).
  *
- *  Related: Array#all?, Array#none?, Array#one?.
+ *  Related: See {Methods for Querying}[rdoc-ref:Array@Methods+for+Querying].
```

Comment:
Nit: all the other sections use a lower case "s" for "see", can we make it consistent?

```suggestion
 *  Related: see {Methods for Querying}[rdoc-ref:Array@Methods+for+Querying].
```